### PR TITLE
Remoting documentation: README and System properties

### PR DESCRIPTION
**Pull request by oleg-nenashev**
- [x] Add README
- [x] Document the system properties
- [x] Annotate the code accordingly

Most likely the settings list is incomplete, I'll add more settings later.

@reviewbybees @jenkinsci/code-reviewers

### CODE REVIEW

**Comment by stephenc** [comment]
:bee: 👍

File: docs/configuration.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+### Available settings
+
+<table>
+  <tbody>
+    <tr>
+      <td>System property</td>
+      <td>Default value</td>
+      <td>[Since](CHANGELOG.md)</td>
+      <td>[Jenkins version(s)](https://jenkins.io/changelog/)</td>
```
Comment (daniel-beck) [inline docs/configuration.md]: GitHub at least can't do inline Markdown within HTML (see 'View File').

Comment (oleg-nenashev) [inline docs/configuration.md]: Yep :(
And it's Table syntax is weird. Converting to HTML

**Comment by oleg-nenashev** [comment]
@daniel-beck fixed it

**Pull request merged**